src/Validation/Validation.php

```diff
@@ -38,6 +38,62 @@ class Validation
      */
     const DEFAULT_LOCALE = 'en_US';
 
+    /**
+     * Same as operator.
+     *
+     * @var string
+     */
+    const COMPARE_SAME_AS = '===';
+
+    /**
+     * Not same as comparison operator.
+     *
+     * @var string
+     */
+    const COMPARE_NOT_SAME_AS = '!==';
+
+    /**
+     * Equal to comparison operator.
+     *
+     * @var string
+     */
+    const COMPARE_EQUAL_TO = '==';
+
+    /**
+     * Not equal to comparison operator.
+     *
+     * @var string
```

Comment:
Those var things are not needed for constants. They are at runtime not changeable, so the type is always clear from the code itself right away.

Reply:
They appear elsewhere.

Reply:
You mean this is done elsewhere? Yeah, but it is really not needed and should be sniffer removed IMO. Duplicating these things for no reason also makes it out of sync error prone at some point and adds additional maintenance overhead.

Reply:
> You mean this is done elsewhere?
Yes.

https://github.com/robertpustulka/cakephp/blob/master/src/Validation/Validator.php#L36

Reply:
Yeah, should all be removed in the future, and no new ones added IMO.